src/ring_theory/polynomial/gauss_lemma.lean

```diff
@@ -125,6 +129,37 @@ begin
   { left,
     apply is_unit_or_eq_zero_of_is_unit_integer_normalization_prim_part f h0.1 h },
 end
+
+lemma is_primitive.dvd_of_fraction_map_dvd_fraction_map {p q : polynomial R}
+  (hp : p.is_primitive) (hq : q.is_primitive) (h_dvd : p.map f.to_map ∣ q.map f.to_map) :
+  (p ∣ q) :=
+begin
+  rcases h_dvd with ⟨r, hr⟩,
+  obtain ⟨⟨s, s0⟩, hs⟩ := @integer_normalization_map_to_map _ _ _ _ _ f r,
```

Comment:
Wasn't the number of explicit args changed, so that we don't need this `@` anymore? (In some recent PR?)

Reply:
I only changed it for the function `localization_map.integer_normalization` itself. I could go through and change it for all of the `theorem`s that I use in this file, if you think that's a good idea.

Reply:
Aah, ok. Well, if this is the only place where you need `@`, then don't bother. But if there are more places where the explicit arg would help, then I think it's a good idea to make those changes. (Maybe not in this PR, though.)